ro_py/groups.py

```diff
@@ -259,7 +259,7 @@ async def get_member_by_id(self, user_id):
         # Find group in list.
         group_data = None
         for group in data['data']:
-            if group['group']['id'] == self.id:
+            if group['group']['id'] == int(self.id):
```

Comment:
This is literally useless

Reply:
You are converting a number to a number.

Reply:
When creating the group object, a string is accepted and is not converted to an int. This just makes sure that if it is a string, then turn it into an int. I do see now however that converting it to an int when making the group object is better and will submit another PR with this removed and doing it directly at the class.